wa/workloads/pcmark/__init__.py

```diff
@@ -17,55 +17,76 @@
 import sys
 import zipfile
 
-from wa import ApkUiautoWorkload
+from wa import ApkUiautoWorkload, Parameter
 from wa.framework.exception import WorkloadError
 
 
 class PcMark(ApkUiautoWorkload):
 
     name = 'pcmark'
+
+    supported_versions = ['3.0.4061', '2.0.3716']
```

Comment:
For the supported versions we probably don't need to be this specific as it looks like things don't tend to break in between versions much so we could get away with just `3` or `2`. The file resolver will still be able to choose more specific versions as appropriate.